pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -201,11 +201,26 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 			return reconcile.Result{}, err
 		}
 
-		// Check account status requeue if we are not ready
 		if byocAccount.Status.State != string(awsv1alpha1.AccountReady) {
+			fmt.Println("ACCOUNT NOT READY")
```

Comment:
Are these debug statements or meant to be shown in the logs? (Line 207 as well)

Reply:
Debug, fixing now.